lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
```

Comment:
@benjamingr I'm not sure what regression you're referring to. These objects are created at startup, not during runtime, so the time to create them doesn't matter as much.